src/cursor.rs

```diff
@@ -27,11 +28,18 @@ struct RawCursorLoading {
 }
 
 impl RawCursor {
-    fn new(response: Response) -> Self {
+    pub(crate) fn new(response: Response) -> Self {
         Self(RawCursorInner::Waiting(response.into_future()))
     }
 
```

Comment:
Probably, we should also add `into_futures03(self) -> RawStream03` (implementing https://docs.rs/futures/latest/futures/prelude/trait.Stream.html).
Probably, under a new `futures03` feature (to support future v1 and `AsyncIterator` from std when they will released).

This is de facto the standard for streams (with built-in [combinators](https://docs.rs/futures/latest/futures/prelude/stream/trait.StreamExt.html), e.g. [concat](https://docs.rs/futures/latest/futures/prelude/stream/trait.StreamExt.html#method.concat)) and should be used for owned types (like `Bytes`).

Alternatively, we can directly implement `Stream` for `RawCursor` (also under the feature).